src/main.js

```diff
@@ -1,8 +1,27 @@
 import Vue from 'vue'
 import App from './App.vue'
+import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+Vue.use(BootstrapVue);
+
+// add vuex to package.json
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  state: {
+    currentview: "landingPage",
```

Comment:
Create a `user: null` in here as well, please. We will keep a persistent user object in here upon successful login.